README.zh-CN.md

```diff
@@ -18,13 +18,20 @@
  - [X] 可扩展空岛领地范围（需要付出一些资源/金钱需要经济插件支持）
  - [X] 可单独创建一个空岛世界与其他世界分离
  - [X] 自定义岛屿重生点或者tp点
- - [X] 利用GriefPrevention的/trust命令来创建一个空岛队伍
+ - [X] 团队功能：
+    - [X] 邀请朋友到岛上
+    - [X] 踢出一个在岛上的玩家
+    - [X] 离开一个岛
+    - [X] 转赠一个岛屿(移交财产么?2333)
```

Comment:
Do these numbers mean something?

Reply:
chinese LOL

Reply:
?2333 is Chinese ??

Reply:
ROFL LOL or oher in laugh

Reply:
What kind of translation are you doing LMAO

Reply:
you add this features let me think it is like: i will dead few mins later,this island give to you and pls take care of it.

Reply:
you add this features let me think it is like: i will dead few mins later,this island give to you and pls take care of it.

Reply:
you add this features let me think it is like: i will dead few mins later,this island give to you and pls take care of it.

Reply:
you add this features let me think it is like: i will dead few mins later,this island give to you and pls take care of it.

Reply:
So you're sharing your delight with your fellow Chinese users? I see :p